x-pack/plugins/beats/server/rest_api/tokens/create.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Joi from 'joi';
+import { get } from 'lodash';
+import { CMServerLibs } from '../../lib/lib';
+import { wrapEsError } from '../../utils/error_wrappers';
+
+const DEFAULT_NUM_TOKENS = 1;
+export const createTokenRoute = (libs: CMServerLibs) => ({
```

Comment:
Nit: should be called `createTokensRoute` (plural) instead since it allows creation of multiple tokens at once?